src/js/views/eva-variant-view.js

```diff
@@ -683,7 +696,12 @@ EvaVariantView.prototype = {
         if(_.isEmpty(variant)){
             var noDataEl = document.querySelector("#summary-grid");
             var noDataElDiv = document.createElement("div");
-            noDataElDiv.innerHTML = '<span>No Data Available</span>';
+            var humanSNPAdditionalInfo = (this.isHumanSNPSearch ?
+                                            'See <a href="' + this.humanSNPLink + '">NCBI data here</a>.'  : '');
+            noDataElDiv.innerHTML = '<span>No Data Available in EVA '
+                                        + (this.accessionID ? 'for ' + this.accessionID + '. ': '')
+                                        + humanSNPAdditionalInfo
+                                        +  '</span>';
             noDataEl.appendChild(noDataElDiv);
             return;
         }
```

Comment:
Check that this [line](https://github.com/EBIvariation/eva-web/pull/163/files#diff-e964ef802b514e19ef5fe7af821932d3R778) is not executed. When using this [URL](http://wwwdev.ebi.ac.uk/eva/EVA1683/?variant&accessionID=rs11362879&species=hsapiens_grch38&assemblyAccession=GCA_000001405.27#population-stats-grid-div) the `navigation-strip` is shown and it shouldn't be

Reply:
Will rectify this...